### PR TITLE
Eslint integration with airbnb extension

### PR DESCRIPTION
**Pull request by ganeshpatro321**
@Harkishen-Singh Please check

#10

### CODE REVIEW

**Comment by Harkishen-Singh** [comment]
@ganeshpatro321 please make the components class-based as was in the original code. They provide easier readability and better management in cases involving a lot of events(which we have planned).

**Comment by ganeshpatro321** [comment]
Have disabled the react/prefer-stateless-function rule in eslint, there are some components which I have modified, lets change them to state components later again.

Lets make sure that everyone has the same eslint configuration file.

**Imp:** Remember to enable this rule later after some adequate amount of development, because unnecessary state components relates to the performance of the app

**Comment by ganeshpatro321** [comment]
Updated! @muskankhedia @Harkishen-Singh

File: src/components/layouts/Sidebar.js [hunk 1]
```diff
@@ -3,36 +3,64 @@ import './layouts.style.css';
...
+          </Link>
+          <Link to="/benchmarks" style={{ textDecoration: 'none' }}>
+            <div>
+              <div className="sidebar-inner">
+                <img src="assets/icons/bench-icon.svg" className="sidebar-inner" alt="Benchmarks" />
+                <div className="sidebar-head sidebar-inner">Benchmarks</div>
+              </div>
+            </div>
+          </Link>
+        </div>
```
Comment (muskankhedia) [inline src/components/layouts/Sidebar.js]: Please fix this. The components- monitoring and benchmarks have been repeated here.

Comment (ganeshpatro321) [reply src/components/layouts/Sidebar.js]: Oops!, might be during resolving the merge conflicts.

Comment (ganeshpatro321) [reply src/components/layouts/Sidebar.js]: Done!

**Pull request merged**